### PR TITLE
Fix for issue 21 and 10

### PR DESCRIPTION
**Pull request by leshkinski**
Closes issue #21 and issue #10

### CODE REVIEW

File: bootstrap/boot.ps1 [hunk 1]
```diff
@@ -1007,7 +1007,37 @@ function Install-PlatformModules
 #########################################################################################################
 #region Helper functions
 #########################################################################################################
+<#
+ Housekeeping function for bootstrap
+#>
+function Start-BootstrapHosekeeping
```
Comment (nick-o) [inline bootstrap/boot.ps1]: This looks like a typo...I'm sure that was supposed to be Start-BootstrapHousekeeping?

File: bootstrap/boot.ps1 [hunk 2]
```diff
@@ -1007,7 +1007,37 @@ function Install-PlatformModules
...
+        # TaskName - name of the boot restart scheduled task
+        [string]$TaskName = 'DSCBoot',
+
+        # Location of bootstrap transcript file
+        [string]$LogPath,
+
+        # DSCAutomation install folder
+        [string]$InstallPath
+    )
+    Write-Verbose "Pereforming boot process housekeeping"
```
Comment (nick-o) [inline bootstrap/boot.ps1]: I know this is a bit pedantic, but there is another typo right there

File: bootstrap/boot.ps1 [hunk 3]
```diff
@@ -1226,6 +1256,8 @@ if ($PullServerConfig)
     {
         Write-Verbose "Error in Pull Server DSC configuration: $($_.Exception)"
     }
+
+    Start-BootstrapHosekeeping -TaskName 'DSCBoot' -LogPath $LogPath -InstallPath $InstallPath -Verbose
```
Comment (nick-o) [inline bootstrap/boot.ps1]: Fix according with the above

File: bootstrap/boot.ps1 [hunk 4]
```diff
@@ -1380,20 +1412,10 @@ else
 
     Write-Verbose "Configuring Client LCM"
     Set-DscLocalConfigurationManager -Path $DSCbootMofFolder -Verbose
+
+    Start-BootstrapHosekeeping -TaskName 'DSCBoot' -LogPath $LogPath -InstallPath $InstallPath -Verbose
```
Comment (nick-o) [inline bootstrap/boot.ps1:1417]: Fix according with the above

**Comment by nick-o** [comment]
Hi Alexei,

Code-wise all good, just a few typos that require fixing. Happy to merge once that has been done.

Cheers
Nico

**Comment by leshkinski** [comment]
Hi @nick-o I fixed typos as per your comments.

**Pull request merged**